web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -62,9 +69,21 @@
   });
 
   onDestroy(() => {
+    if (browser) document.removeEventListener('keydown', handleKeyboardPress);
     assetStore.setInitialState(0, 0, { totalCount: 0, buckets: [] }, undefined);
   });
 
+  const handleKeyboardPress = (event: KeyboardEvent) => {
+    if (event.key === '/') event.preventDefault();
+    if (!$isViewingAssetStoreState) {
+      switch (event.key) {
+        case '/':
```

Comment:
Should prevent default happen here instead?

Reply:
When viewing an asset on Google Photos, the shortcut key '/' is disabled. By implementing the prevent default here, you can ensure that the quick find shortcut will never be triggered while navigating through the asset grid or while viewing an asset.

Reply:
Sorry if I wasn't clear, I was speaking about line 77 😅